utilities/folder.py

```diff
@@ -31,11 +32,11 @@ def __init__(self, data):
         self.sync()
 
     def sync(self):
+        print("Syncing:", self.name)
+
         # TODO:
         # This need to be asyc
-        from utilities.sync_core import SyncCore
```

Comment:
Oh no no byczq

Reply:
![image](https://user-images.githubusercontent.com/41760168/134785867-77a3e574-6ddc-489c-8f31-f7b7a3b208ee.png)

Reply:
why you delete it